collector/lib/ProfilerHandler.cpp

```diff
@@ -13,7 +13,7 @@
 
 namespace collector {
 
-const std::string ProfilerHandler::kCPUProfileFilename = "/module/cpu_profile";
+const std::string ProfilerHandler::kCPUProfileFilename = "/var/profiles/cpu_profile";
```

Comment:
Maybe worth adding a few lines of documentation about that?

Reply:
Makes sense, I'll add a comment.

Reply:
I also added a small section to our docs, think that should be enough.